lib/LedgerSMB/Sysconfig.pm

```diff
@@ -422,6 +422,18 @@ if(!(-d LedgerSMB::Sysconfig::tempdir())){
 sub check_permissions {
     use English qw(-no_match_vars);
 
+    if($EUID == 0){
+        die_pretty( "Running LedgerSMB as root is a security problem",
```

Comment:
While true, can we make this sound as if this is a general security probem and not specifically LedgerSMB related?